docs/Using-Fleet/FAQ.md

```diff
@@ -182,10 +189,41 @@ Fleet relies on UUIDs so any overlap with host IP addresses should not cause a p
 
 Yes, Orbit can be run alongside osquery. The osquery instance that Orbit runs uses its own database directory that is stored within the Orbit directory.
 
+## Can I disable auto-updates for Orbit?
+
+Yes, auto-updates can be disabled by passing `--disable-updates` as a flag when running `fleetctl package` to generate your installer (easy) or by deploying a modified systemd file to your hosts (more complicated). We'd recommend the flag:
+
+```
+fleetctl package --fleetctl package --type=deb --fleet-url=https://localhost:8080 --enroll-secret=superRandomSecret --disable-updates
+```
+## Can I bundle osquery extensions into Orbit?
+
+This isn't supported yet, but we're working on it! 
+
 ## What happens to osquery logs if my Fleet server or my logging destination is offline?
 
 If Fleet can't send logs to the destination, it will return an error to osquery. This causes osquery to retry sending the logs. The logs will then be stored in osquery's internal buffer until they are sent successfully, or they get expired if the `buffered_log_max`(defaults to 1,000,000 logs) is exceeded. Check out the [Remote logging buffering section](https://osquery.readthedocs.io/en/latest/deployment/remote/#remote-logging-buffering) on the osquery docs for more on this behavior.
 
+## How does Fleet work with osquery extensions?
+
+Any extension table available in a host enrolled to Fleet can be queried by Fleet. Note that the "compatible with" message may show an error because it won't know your extension table, but the query will still work, and Fleet will gracefully ignore errors from any incompatible hosts.
+
+## Why do I see "Unknown Certificate Error" when adding hosts to my dev server?
```

Comment:
@ksatter do you feel we answered this question or did we show how to avoid this message from popping up?

Reply:
Since it's specifically related to dev servers. I feel we're good.